services/parser-database/connector.py

```diff
@@ -74,7 +74,7 @@ def save_keywords_in_memory(keywords, article):
             keywords_in_memory[keyword] = []
         keywords_in_memory[keyword].append({
             "number": article["number"],
-            "id": article["articleNumber"],
+            "id": article["id"],
             "frequency": frequency
```

Comment:
Shouldn't this have broken some tests?

Reply:
Just ran them over again and everything passed 👍

Reply:
Was there a failing test before you changed this? If not, there is a unit test missing and should be added in this PR to avoid a similar breakage in the future.

Reply:
Hmm, I assume the unit tests are mocking this out correctly, but the actual service was not returning "id". As long as the unit tests are using "id" also instead of "articleNumber", I'm fine with this.

Reply:
I don't think anything is being mocked to test this function -- it should be a pretty simple test and an obvious failure

Reply:
Okay, let me add the test